src/Mikrofrontend.tsx

```diff
@@ -1,35 +1,45 @@
-import useSWRImmutable from 'swr/immutable'
-import { fetcher } from './api/api'
-import { apiUrl } from './api/urls'
-import { Pamelding } from './pamelding/Pamelding'
-import { MAL_TYPE_ANNET } from './utils'
+import {Pamelding} from './pamelding/Pamelding'
+import {MAL_TYPE_ANNET} from './utils'
+import {useEffect, useState} from 'react'
+import {createPamelding} from './api/api.ts'
+import {PameldingResponse} from './api/data/pamelding.ts'
 
 const Mikrofrontend = () => {
-  const { data } = useSWRImmutable(apiUrl, fetcher)
+  const [info, setInfo] = useState<PameldingResponse | null>(null)
+
+  useEffect(() => {
+    createPamelding({
+      deltakerlisteId: '6b6578eb-eae0-4ad7-8a69-79db3cea4f64',
+      personident: '12345678901'
+    })
+      .then(data => setInfo(data))
+  }, [])
+
+  if (info === undefined) return (<div>Ikke lastet</div>)
```

Comment:
Dette er sikkert bare midlertidig kode men:

`info` kan vel ikke være `undefined` her, for du setter den til `null` over?

Er det ikke bedre å la info være `undefined` by default?